docs/references/rest-apis/rest-position-api.md

```diff
@@ -0,0 +1,84 @@
+!!! note
+
+    This API can also be accessed via the RequestHandler with app-id: `POS-V1`.
+
+#### Get Position
+- Method: GET
+- API PATH: `/services/position/v1/position/`
+
+##### Responses
+- 200 OK status
+
+```JSON
```

Comment:
What is the response if we have no fix? Are all fields always guaranteed to be present?